waku/v2/protocol/filter/client.go

```diff
@@ -287,7 +302,7 @@ func (wf *WakuFilterLightNode) request(ctx context.Context, requestID []byte,
 
 	}
 
-	if filterSubscribeResponse.RequestId != request.RequestId {
+	if filterSubscribeResponse.RequestId != "N/A" && filterSubscribeResponse.RequestId != request.RequestId {
```

Comment:
what is "N/A"? don't remember seeing it before.

also is this implemented in nwaku as well? otherwise this condition will fail if filter server is nwaku node.